Tasks/intersection.js

```diff
@@ -1,25 +1,16 @@
 // Find an intersection of two dictionaries
 
-intersection = function intersection(object_1, object_2) {
-  first_keys = Object.keys(object_1);
-  {
-  }
-  for (attribute_name of first_keys) {
-    if (object_1[attribute_name] === object_2[attribute_name]) {
-      object_2[attribute_name] = object_1[attribute_name];
-      {
-      }
-    } else {
-      delete object_1[attribute_name];
-      {
-      }
-    }
-    {
+'use strict';
+
+const intersection = (firstObj, secondObj) => {
+  const result = {};
+  const firstObjKeys = Object.keys(firstObj);
+  for (const key of firstObjKeys) {
+    if (firstObj[key] === secondObj[key]) {
+      result[key] = firstObj[key];
```

Comment:
Use intermediate variable to store value of expression `firstObj[key]` instead of double calculation